### DISCUSSION TITLE
Extend the docs (blog?) for the Rust Actix template to provide next steps

### DISCUSSION THREAD

**Discussion by dupuyarc** [noop-inc/documentation#124] [Feature Requests and Ideas]
Just for kicks, I thought I would try this one first. I was able to deploy the template from the Desktop Workshop, I could see that things started running, but wasn't sure what was there. On the Application page, I saw the endpoint [`template-rust-actix.local.noop.app`](https://template-rust-actix.local.noop.app/) displayed under Active Endpoints (and after clicking through the Endpoint page and the Authority link, I got this page:

<img width="570" alt="Screenshot 2024-02-27 at 16 57 33" src="https://github.com/noop-inc/documentation/assets/157545233/548dd8cd-a580-408c-a11e-fb5bd55eac0c">

In a terminal, I could generate the GET alright, but it sort of left me at a loss as far as how I could proceed, since it just returned `[]` (presumably an empty JSON list), and I had no idea what sort of data one should post to the `/users/add` page.

<img width="650" alt="Screenshot 2024-02-27 at 16 59 08" src="https://github.com/noop-inc/documentation/assets/157545233/a4a83b7c-2f59-44e6-85dd-24befb621677">

It's a pretty far cry from the equivalent Django page (below). Obviously this isn't a problem with Noop or the templates, but it would help to have something in the [docs](https://github.com/noop-inc/template-rust-actix/blob/main/README.md) or the [blog](https://noop.dev/blog/launch-rust-app/) indicating that this is what's expected from an unmodified Actix + Postgres deployment.
(There's another small typo in the blog: "OTo start a new Actix project").

Perhaps a link to [Actix docs](https://actix.rs/docs/getting-started/) might help, although that describes a much more minimal deployment without Postgres, so isn't really so useful here. Almost anything that gives you some idea about what you need to do to extend this web server + Postgres DB deployment would be useful. Even a `curl` command line snippet for generating a POST to `/users/add` would be a start.

<img width="1140" alt="Screenshot 2024-02-27 at 16 57 07" src="https://github.com/noop-inc/documentation/assets/157545233/490b8572-5550-4283-9f11-4f7e661f662e">

**Comment by 1800joe** [comment]
Excellent points. We'll put this on our list!

Thanks!
